invenio_app_rdm/theme/views.py

```diff
@@ -94,3 +96,26 @@ def help_statistics():
             "invenio_app_rdm/help/statistics.en.html",
         ]
     )
+
+
+def notification_settings():
+    """View for notification settings."""
+    preferences_notifications_form = NotificationsForm(
+        formdata=None, obj=current_user, prefix="preferences-notifications"
+    )
+
+    # Pick form
+    form_name = request.form.get("submit", None)
+    form = preferences_notifications_form if form_name else None
+
+    # Process form
+    if form:
+        form.process(formdata=request.form)
+        if form.validate_on_submit():
+            handle_notifications_form(form)
+            return redirect(url_for("invenio_notifications_settings.index"), code=303)
```

Comment:
minor: shouldn't we just render the same page on success with a message flash?

(see also https://github.com/inveniosoftware/invenio-users-resources/pull/77/files#r1212195098)

Reply:
removed the redirect